### DISCUSSION TITLE
RNAV approaches on MCDU

### DISCUSSION THREAD

**Discussion by DumbSheep123** [flybywiresim/aircraft#6241] [Ideas]
Hello developers at FBWS,

I know you are busy so I would like to keep this short. ILS approaches are great, but not every airport has them. I suggest to add RNAV support for the MCDU.

Remember to always eat 🍪's,
DumbSheep123

**Comment by andrejetcat** [comment]
hey DumbSheep123  

look at their facebook page... :) RNAV comming soon!
